doc/cprover-manual/smt2-incr.md

```diff
@@ -0,0 +1,40 @@
+[CPROVER Manual TOC](../)
+
+## Incremental SMT backend
+
+CBMC comes with a solver-agnostic incremental SMT backend.
+
+The SMT incremental backend supports the following `C` primitives:
```

Comment:
We should probably refer to them as "major features" or just "features", not primitives, IMO.

Reply:
changed